1-js/02-first-steps/15-function-expressions/article.md

```diff
@@ -121,24 +121,24 @@ function ask(question, yes, no) {
 */!*
 
 function showOk() {
-  alert( "You agreed." );
+  alert( "Może być." );
 }
 
 function showCancel() {
-  alert( "You canceled the execution." );
+  alert( "Nie chcesz nic wywoływać." );
 }
 
-// usage: functions showOk, showCancel are passed as arguments to ask
-ask("Do you agree?", showOk, showCancel);
+// użycie: funkcje showOk, showCancel są przekazywane do funkcji ask jako argumenty
+ask("Zgadzasz się?", showOk, showCancel);
 ```
 
-In practice, such functions are quite useful. The major difference between a real-life `ask` and the example above is that real-life functions use more complex ways to interact with the user than a simple `confirm`. In the browser, such function usually draws a nice-looking question window. But that's another story.
+Okazuje się, że takie funkcje są całkiem przydatne. W przeciwieństwie do powyższego przykładu, funkcje wykorzystywane w praktyce używają znacznie bardziej skomplikowanych sposobów interakcji z użytkownikiem niż zwykłe `confirm`. W przeglądarce, taka funkcja zazwyczaj rysuje ładnie wyglądające okienko, chociaż jest to zupełnie inna historia.
```

Comment:
Replace `W przeglądarce, taka funkcja zazwyczaj rysuje ładnie wyglądające okienko` by `W przeglądarce, taka funkcja zazwyczaj rysuje ładnie wyglądające okienko z pytaniem`. Don't lose the context of the original sentence :)